Add deprecation notices for Felix configuration paramaters (#1316)
tigera/docs: calico_versioned_docs/version-3.27/release-notes/index.mdx

 - eBPF: kube-proxy ServiceInternalTrafficPolicy is now GA and setting the gate would generate a warning message. [calico #8242](https://github.com/projectcalico/calico/pull/8242) (@tomastigera)
 - eBPF: fixed policy cleanup after felix restart if a device is not present anymore [calico #8231](https://github.com/projectcalico/calico/pull/8231) (@fasaxc)

## Deprecated and removed features

* In {{prodname}} 3.27, the Felix configuration parameter `BPFConnectTimeLoadBalancingEnabled` is deprecated and is planned to be removed in a future release.

>   As an alternative, you can use the `bpfConnectTimeLoadBalancing` configuration parameter.

Reviewer: ```suggestion
  Use the `bpfConnectTimeLoadBalancing` and `BPFHostNetworkedNATWithoutCTLB` configuration parameters instead.
```

Author: Changed, thanks.
